EIPS/eip-5725.md

```diff
@@ -0,0 +1,189 @@
+- `vestedPayout()` will provide the total amount of tokens which are eligible for release (including claimed tokens), while `claimablePayout()` provides the amount of tokens which can be claimed at the current `timestamp`.
+- `vestedPayoutAtTime()` provides functionality to iterate through the `vestingPeriod()` and provide a visual of the release curve. This allows for tools to iterate through a vesting schedule and create a visualization using on-chain data. It would be incredible to see integrations that create SVG visuals of vesting curves directly on-chain. 
+
+
+
+**timestamps**
+Generally in Solidity development it is advised against using `block.timestamp` as a state dependant variable as the timestamp of a block can be manipulated by a miner. The choice to use a `timestamp` over a `block` is to allow the interface to work across multiple **Ethereum Virtual Machine** (EVM) compatible networks which generally have different block times. Block proposal with a significantly fabricated timestamp will generally be dropped by all node implementations which makes the window for abuse negligible.
+
+The `timestamp` makes cross chain integration easy, but internally, the reference implementation keeps track of the token payout per Vesting NFT to ensure that excess tokens alloted by the vesting terms cannot be claimed. 
+
+**limitation of scope**
+
+The standard does not implement the following features:
+- Vesting Curves
+- Rental
+- Beneficiary
+
+This is done intentionally to keep the base standard simple. These features can and likely will be added through extensions of this standard.
+
+## Backwards Compatibility
+<!-- All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright. -->
+- The Vesting NFT standard is meant to be fully backwards compatible with any current [EIP-721 standard](./eip-721.md) integrations and marketplaces.
+- The Vesting NFT standard also supports [EIP-165 standard](./eip-165.md) interface detection for detecting `EIP-721` compatibility, as well as Vesting NFT compatibility.
+
+## Test Cases
+The reference vesting NFT repository includes tests written in Hardhat.
+
+## Reference Implementation
+<!-- An optional section that contains a reference/example implementation that people can use to assist in understanding or implementing this specification.  If the implementation is too large to reasonably be included inline, then consider adding it as one or more files in `../assets/eip-####/`. -->
+A reference implementation of this EIP can be found in [this repository](../assets/eip-5725/contracts/).
```

Comment:
This is causing CI to fail. You may need a README of sorts there or link to a specific file.

Reply:
Thanks for the input, appreciate it! We weren't sure why this one was failing but we'll add a README in the assets subdir.

Reply:
Added a README. :)